src/main/scala/com.snowplowanalytics.sauna/Sauna.scala

```diff
@@ -0,0 +1,36 @@
+package com.snowplowanalytics.sauna
+
+import java.io.File
+
+import com.typesafe.config.ConfigFactory
+
+import com.snowplowanalytics.sauna.responders.optimizely._
+import com.snowplowanalytics.sauna.watchers._
+
+case class SaunaConfig(queueName: String, accessKeyId: String,
+                       secretAccessKey: String, optimizelyToken: String, saunaRoot: String)
+
+object Sauna extends App {
+  if (args.length != 1) {
+    println("""Usage: 'sbt "run <path_to_file_with_credentials>"' """)
+    System.exit(1)
+  }
+
+  private val conf = ConfigFactory.parseFile(new File(args(0)))
+  val saunaConfig = SaunaConfig(
+    conf.getString("queue.name"),
+    conf.getString("aws.access_key_id"),
+    conf.getString("aws.secret_access_key"),
+    conf.getString("optimizely.token"),
+    conf.getString("sauna.root")
+  )
```

Comment:
Move this ^^^ into a function and then create a new test to verify the hocon can be successfully parsed from a string.